src/Operators/vorticity_operators.jl

```diff
@@ -30,11 +22,20 @@ end
 """
      Γᶠᶠᶜ(i, j, k, grid, u, v)
 
- The vertical circulation associated with horizontal velocities ``u`` and ``v``.
- """
-@inline Γᶠᶠᶜ(i, j, k, grid::OrthogonalSphericalShellGrid, u, v) = 
-    ifelse(on_south_west_corner(i, j, grid) | on_north_west_corner(i, j, grid),
-           Δy_qᶜᶠᶜ(i, j, k, grid, v) - Δx_qᶠᶜᶜ(i, j, k, grid, u) + Δx_qᶠᶜᶜ(i, j-1, k, grid, u), 
-           ifelse(on_south_east_corner(i, j, grid) | on_north_east_corner(i, j, grid),
-                  - Δy_qᶜᶠᶜ(i-1, j, k, grid, v) - Δx_qᶠᶜᶜ(i, j, k, grid, u) + Δx_qᶠᶜᶜ(i, j-1, k, grid, u), 
-                  δxᶠᵃᵃ(i, j, k, grid, Δy_qᶜᶠᶜ, v) - δyᵃᶠᵃ(i, j, k, grid, Δx_qᶠᶜᶜ, u)))
\ No newline at end of file
+The vertical circulation associated with horizontal velocities ``u`` and ``v``.
+"""
+
+@inline function Γᶠᶠᶜ(i, j, k, grid::OrthogonalSphericalShellGrid, u, v)
+    Hx, Hy = grid.Hx, grid.Hy
+    Γ = ifelse((i == 1 - Hx) | (j == 1 - Hy),
+               NaN,
```

Comment:
Putting NaN's can slow the model down, are we sure we want to do this?

Reply:
I was using NaNs to aid in visualization, but they aren't necessary. Let me go ahead and remove them to avoid any potential slowdown.